src/libraries/Native/Unix/System.Security.Cryptography.Native/entrypoints.c

```diff
@@ -144,10 +146,12 @@ static const Entry s_cryptoNative[] =
     DllImportEntry(CryptoNative_EvpMdCtxDestroy)
     DllImportEntry(CryptoNative_EvpMdSize)
     DllImportEntry(CryptoNative_EvpPkeyCreate)
+    DllImportEntry(CryptoNative_EvpPKeyCreateRsa)
     DllImportEntry(CryptoNative_EvpPKeyCtxCreate)
     DllImportEntry(CryptoNative_EvpPKeyCtxDestroy)
     DllImportEntry(CryptoNative_EvpPKeyDeriveSecretAgreement)
     DllImportEntry(CryptoNative_EvpPkeyDestroy)
+    DllImportEntry(CryptoNative_EvpPKeyDuplicate)
     DllImportEntry(CryptoNative_EvpPkeyGetDsa)
     DllImportEntry(CryptoNative_EvpPkeyGetEcKey)
     DllImportEntry(CryptoNative_EvpPkeyGetRsa)
```

Comment:
You got rid of `CryptoNative_EvpPkeyGetRsa` on managed side. Should it be removed here too?

Ditto for `CryptoNative_EvpPkeySetRsa` (not visible in the GitHub diff).

Reply:
Hm, thought I had.  But apparently they're in pal_evp_pkey_rsa, not pal_rsa.  Makes sense.  I'll tidy this up when I fix the OpenSSL 1.0 build breaks.  :smile: